utils/cronjobs_osgeo_lxd/cron_grass_current_stable_build_binaries.sh

```diff
@@ -8,36 +8,34 @@
 ###################################################################
 # how it works:
 # - it updates locally the GRASS source code from github server
-# - configures, compiles
+# - configures source code and then compiles it
 # - packages the binaries
 # - generated the install scripts
 # - generates the pyGRASS 8 HTML manual
 # - generates the user 8 HTML manuals
 # - injects DuckDuckGo search field
 
 # Preparations, on server (neteler@grasslxd:$):
-#  - Install PROJ incl Datum shift grids
-#  - Install GDAL
-#  - Install apt-get install texlive-latex-extra python3-sphinxcontrib.apidoc
-#  - Clone source from github:
-#    mkdir -p ~/src ; cd ~/src
-#    git clone https://github.com/OSGeo/grass.git releasebranch_8_4
-#    cd releasebranch_8_4
-#    git checkout releasebranch_8_4
-#  - Prepare target directories:
-#    cd /var/www/code_and_data/
-#    mkdir grass84
-#    cd /var/www/html/
-#    ln -s /var/www/code_and_data/grass84 .
+# - Install PROJ incl Datum shift grids
+# - Install GDAL
+# - Install apt-get install texlive-latex-extra python3-sphinxcontrib.apidoc
+# - install further dependencies:
+#     cd $HOME/src/releasebranch_8_4/ && git pull && sudo apt install $(cat .github/workflows/apt.txt)
+# - run this script
+# - one time only: cross-link code into web space on grasslxd server:
+#     cd /var/www/html/
+#     ln -s /var/www/code_and_data/grass84 .
```

Comment:
How does this relate to instructions in the comment in the PR? A single clear place is needed, maybe in readme, but here is good if it needs to be different for different scripts. (But is it?)

Reply:
It is partially different due to the different versions.

I'll try to clean this up.

Reply:
We do have different branches in the main repo and this is already using them, so perhaps that's one way how to make it different, yet same on the level of these scripts. In grass-addons we also have different branches, so other things can be resolved through that if needed, but that seems less applicable right now.

Reply:
And instructions simplified in cbdf786dddc960d6e2d06b61df6e7f365b946d17